.github/workflows/Android-CI.yml

```diff
@@ -0,0 +1,20 @@
+name: PullRequest
+
+on: [pull_request]
+
+jobs:
+  buildTest:
+    name: Build SDK and HelloCardboard for Android
+    runs-on: macOS-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.2
+        with:
+          fetch-depth: 0
+      - name: Build sdk
+        run: ./gradlew :sdk:assemble
+      - name: Code checks
```

Comment:
Move this step below `Build HelloCardboard`

Reply:
See https://docs.gradle.org/current/userguide/java_plugin.html

Reply:
I tried this. But the check fails on the Build sdk.
see: https://github.com/arilow/cardboard/actions/runs/664838625

Reply:
I think we can skip the first linked error. It was a network one. For the second case it seems like the link is wrong.